tests/test_utils.py

```diff
@@ -250,3 +250,28 @@ def test_win_normalize_drive(self, input_path, expected):
     @pytest.mark.skipif(os.name == 'nt', reason='*nix file paths tested')
     def test_nix_normalize_drive(self, input_path, expected):
         assert pipenv.utils.normalize_drive(input_path) == expected
+
+    @pytest.mark.requirements
+    def test_get_requirements(self):
```

Comment:
Can we get some divisions into conceptual testing blocks and minor comments for what we're confirming/refuting here?

Reply:
done.